index.js

```diff
@@ -275,6 +276,16 @@ export default class WorkspacesPlugin extends Plugin {
       }
     };
 
+    /**
+     * In workflows where publishing is handled on C.I.,
+     * and not be release-it, the environment will often require
+     * that the lockfile be updated -- this is usually done by
+     * running the install command for the package manager.
+     */
+    if (await findUp('pnpm-lock.yaml')) {
+      await this.exec(`pnpm install`);
```

Comment:
this was added as it was found to be a required operation when using a patch of a previous version of this PR.

if we don't run install / update the lockfile before commit, CI fails.

This occurs when using CI-based publishing and any of the following:
- versions are regular semver ranges, as bumped by this workspace plugin
- versions workspace:protocol ranges end up needing to be updated as a part of version bump

Reply:
I think that this isn't limited to `pnpm`. The same is true in `npm` as far as I can tell. I think the only one it isn't true for is `yarn`?

Reply:
should this run install for any package manager? / all of them? would it then need to be configurable?

Reply:
Right now I don't think we do any package manager detection other than pnpm, so in order to always run install across the board we'd have to actually do that.

Let's punt on that for now (we can do that in a follow up PR)